keras_nlp/models/bert.py

```diff
@@ -25,20 +25,54 @@ def _bert_kernel_initializer(stddev=0.02):
     return keras.initializers.TruncatedNormal(stddev=stddev)
 
 
+def _handle_weights_and_vocab_size(bert_variant, weights, vocabulary_size):
```

Comment:
small docstring, since this is a little complex?

"""
Lookup pretrained defaults for weights and vocabulary_size.

The helper will validate the weights and vocabulary_size argument, and fully
resolve them in the case we are loading pretrained weights.
"""